grammars/html.cson

```diff
@@ -587,12 +587,69 @@
         'include': '#unquoted-attribute'
       }
     ]
+  'tag-class-attribute':
+    'begin': '\\b(class)\\b\\s*(=)'
+    'captures':
+      '1':
+        'name': 'entity.other.attribute-name.class.html'
+      '2':
+        'name': 'punctuation.separator.key-value.html'
+    'end': '(?<=\'|")|(?=\\s|>)'
+    'name': 'meta.attribute-with-value.class.html'
+    'patterns': [
+      {
```

Comment:
Just include `#string-double-quoted`, `#string-single-quoted`, and `#unquoted-attribute` instead.

Reply:
I am not sure if the change I have made here is correct, probably I did not understand it well.

With this change the class value is not wrapped with an `html` tag.